sampleapp/src/main/java/org/hyperledger/identus/walletsdk/sampleapp/Sdk.kt

```diff
@@ -61,7 +61,9 @@ class Sdk {
 
         (pluto as PlutoImpl).start(context)
         agent.start()
-        agent.startFetchingMessages()
+        try {
+            agent.startFetchingMessages()
+        } catch (_: Exception) {}
```

Comment:
why are we swallowing exceptions here?

Reply:
This is the sample app, the user of our SDK will decide how to handle exceptions here whether known or unknown.